### PR TITLE
Feat: Add time window for GetTrace of anonymizer

### PR DESCRIPTION
**Pull request by rim99**
## Which problem is this PR solving?
- Part of #4150

## Description of the changes
- Add optional time window for GetTrace of anonymizer

## How was this change tested?
- unit test

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits
- [x] I have added unit tests for the new functionality
- [x] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `npm run lint` and `npm run test`

### CODE REVIEW

File: cmd/anonymizer/app/flags.go [hunk 1]
```diff
@@ -72,6 +76,16 @@ func (o *Options) AddFlags(command *cobra.Command) {
 		maxSpansCount,
 		-1,
 		"The maximum number of spans to anonymize")
+	command.Flags().Int64Var(
+		&o.StartTime,
+		startTime,
+		-1,
```
Comment (yurishkuro) [inline cmd/anonymizer/app/flags.go:82]: why not 0?

Comment (rim99) [reply cmd/anonymizer/app/flags.go]: Yes, 0 makes things easier

File: cmd/anonymizer/main.go [hunk 2]
```diff
@@ -53,7 +54,9 @@ func main() {
 				logger.Fatal("error while creating query object", zap.Error(err))
 			}
 
-			spans, err := query.QueryTrace(options.TraceID)
+			startTime := time.Unix(0, options.StartTime)
```
[the remark this answers is not in the document]
Comment (rim99) [reply cmd/anonymizer/main.go:57]: Changed default value as 0, so no need to check then

[the remark this answers is not in the document]
Comment (yurishkuro) [reply cmd/anonymizer/main.go:57]: still need to check, time 0 is not the same as epoch 0

[the remark this answers is not in the document]
Comment (rim99) [reply cmd/anonymizer/main.go]: Done

[the remark this answers is not in the document]
Comment (rim99) [reply cmd/anonymizer/main.go]: Done

[the remark this answers is not in the document]
Comment (rim99) [reply cmd/anonymizer/main.go]: Done

File: cmd/anonymizer/main.go [hunk 3]
```diff
@@ -93,3 +98,13 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func initTime(ts int64) time.Time {
+	var t time.Time
+	if ts != 0 {
+		t = time.Unix(0, ts)
+	} else {
+		t = time.Time{}
```
Comment (yurishkuro) [inline cmd/anonymizer/main.go:107]:
```suggestion
```
var declaration already does this

**Review by yurishkuro — APPROVED** [review]

**Review by yurishkuro — APPROVED** [review]

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**